test/addons-napi/test_constructor/test.js

```diff
@@ -22,9 +22,9 @@ const propertyNames = [];
 for (const name in test_object) {
   propertyNames.push(name);
 }
-assert.ok(propertyNames.indexOf('echo') >= 0);
-assert.ok(propertyNames.indexOf('readwriteValue') >= 0);
-assert.ok(propertyNames.indexOf('readonlyValue') >= 0);
+assert.ok(propertyNames.includes('echo'));
+assert.ok(propertyNames.includes('readwriteValue'));
+assert.ok(propertyNames.includes('readonlyValue'));
 assert.ok(propertyNames.indexOf('hiddenValue') < 0);
 assert.ok(propertyNames.indexOf('readwriteAccessor1') < 0);
 assert.ok(propertyNames.indexOf('readwriteAccessor2') < 0);
```

Comment:
These should be updated to `!propertyNames.includes()` too so that the code is consistent.